arches/app/models/graph.py

```diff
@@ -779,6 +779,7 @@ def flatten_tree(tree, node_id_list=[]):
             copy_of_self.subtitle = ""
             copy_of_self.description = ""
             copy_of_self.author = ""
+            copy_of_self.serialized_graph = ""
```

Comment:
I thought about this a little since we last spoke. Since we're cloning the graph reason stands that we're also to clone `published` state. So instead of just nulling out serialized_graph, we should ensure the cloned graph gets saved to the database. After the save we should then re-publish the graph if the original graph was published.

Reply:
deferred to 3rd developer -- the cloned graph should be unpublished